tests/test_verifylib.py

```diff
@@ -395,26 +395,45 @@ def setUp(self):
     self.materials = {
       "foo": {"sha256": self.sha256_foo},
       "foobar": {"sha256": self.sha256_foobar},
+      "sub/foobar": {"sha256": self.sha256_foobar},
       "sub/foo": {"sha256": self.sha256_foo},
-      "sub/foobar": {"sha256": self.sha256_foobar}
-
+      "lib/build/foobar": {"sha256": self.sha256_foobar},
+      "lib/foo": {"sha256": self.sha256_foo},
+      "lib/build/foo": {"sha256": self.sha256_foo},
+      "lib/foobar": {"sha256": self.sha256_foobar},
+      "build/lib": {"sha256": self.sha256_foobar},
+      
     }
     self.products = {
       "bar": {"sha256": self.sha256_bar},
       "barfoo": {"sha256": self.sha256_barfoo},
+      
+      "lib": {"sha256": self.sha256_bar},
```

Comment:
We're treating `lib` as a directory, so it shouldn't be recorded with a hash on its own.

Reply:
In this pr, this change will appear as I commented this line out, but in the next change will show that I removed this line from the code.